saltcellar/saltcellar/environment.py

```diff
@@ -0,0 +1,98 @@
+#
+# Copyright 2018 Red Hat, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+"""Gets system environment data."""
+
+import logging
+import os
+import platform
+import subprocess
+import sys
+
+# pylint: disable=invalid-name
+# Get an instance of a logger
+logger = logging.getLogger(__name__)
+
+
+def commit():
```

Comment:
how about something like get_commit_info? `commit` makes me think it performs a commit.